task_10/lev.brekhov/homework10/src/main/com/hillel/WordPalindrome.java

```diff
@@ -0,0 +1,34 @@
+package com.hillel;
+
+import java.util.Scanner;
+
+public class WordPalindrome {
+
+  public static void main(String[] args) {
+    Scanner sc = new Scanner(System.in);
+    String str = sc.nextLine();
+    String[] arrayStrings = str.split(",");
+    int count = 0;
+    boolean isPalindrome;
```

Comment:
It is better to define variable in its own scope. You don't need it outside for loop.